src/Common/ErrorCodes.cpp

```diff
@@ -602,6 +602,7 @@
     M(721, DEPRECATED_FUNCTION) \
     M(722, ASYNC_LOAD_WAIT_FAILED) \
     M(723, PARQUET_EXCEPTION) \
+    M(724, TOO_LONG_TABLE_NAME) \
```

Comment:
I don't think we need a separate error code for this, it's too specific. We can use `ARGUMENT_OUT_OF_BOUND`